back/models/game.js

```diff
@@ -7,7 +8,12 @@ class Game {
     this.players = [];
     this.deck = shuffle(createDeck());
     this.activePlayerId = null;
-    this.cardStack = null;
+    this.cardStack = [];
+  }
+
+  getPlayerById(playerId) {
+    const player = this.players.find(player => player.id === playerId);
+    return player;
```

Comment:
variable is redundant, we could return this:
`return this.players.find(player => player.id === playerId);`